gnomad_qc/v4/annotations/vrs_annotation_batch.py

```diff
@@ -0,0 +1,523 @@
+"""
+This is a batch script which adds VRS IDs to a Hail Table by creating sharded VCFs, running a vrs-annotation script on each shard, and merging the results into the original Hail Table.
+
+Advise to run from a backend (eg. hailctl batch submit) to avoid losing progress
+ in case of disconnection.
+
+Example command to use `hailctl batch submit`:
+hailctl batch submit \
+--image-name us-central1-docker.pkg.dev/broad-mpg-gnomad/images/vrs084 \
+gnomad_qc/gnomad_qc/v4/annotations/vrs_annotation_batch.py \
+-- \
+--billing-project gnomad-annot \
+--working-bucket gnomad-tmp-4day \
+--image us-central1-docker.pkg.dev/broad-mpg-gnomad/images/vrs084 \
+--version test_v4.0_exomes \
+--prefix v4 \
+--header-path gs://gnomad/v4.0/annotations/exomes/vrs-header-fix.txt \
+--run-vrs \
+--annotate-original \
+--overwrite \
+--backend-mode batch \
+--test
+"""
```

Comment:
all references to "version" and "prefix" can now be removed from this script

Reply:
even from arguments or just in the example?

Reply:
yeah also from the arguments